ts/IterableInboxMessageCell.tsx

```diff
@@ -139,17 +137,146 @@ const defaultMessageListLayout = (
    )
 }
 
-const IterableInboxMessageListItem = ({
+type MessageCellProps = {
+   key: string,
```

Comment:
Can you remove unused properties as a result of this consolidation? `key`, at least, isn't used anywhere else.

Reply:
Yes, thanks.